### PR TITLE
Translation: update zh_CN

### PR DESCRIPTION
**Pull request by cyy5358**
1. Remove 'fuzzy' tag where translation is good enough.
2. Remove trailing spaces.
3. Update translation for sentence
```text
"Remove tag \"%s\" and its descendant tags from all notes?"
```
4. Add translation for sentences
```text
"Read time: %s min"
"Tag name cannot start or end with a `/`."
"Tag name cannot contain `//`."
```

### CODE REVIEW

**Comment by cyy5358** [comment]
1. Remove 'fuzzy' tag where translation is good enough.
2. Remove trailing spaces.
3. Update translation for sentence
```text
"Remove tag \"%s\" and its descendant tags from all notes?"
```
4. Add translation for sentences
```text
"Read time: %s min"
"Tag name cannot start or end with a `/`."
"Tag name cannot contain `//`."
```

File: CliClient/locales/zh_CN.po [hunk 1]
```diff
@@ -897,9 +898,9 @@ msgid "Delete"
 msgstr "删除"
 
 #: ElectronClient/gui/SideBar/SideBar.min.js:276
-#, fuzzy, javascript-format
+#, javascript-format
 msgid "Remove tag \"%s\" and its descendant tags from all notes?"
-msgstr "从所有笔记中删除标签 \"%s\" ？"
+msgstr "从所有笔记中删除标签 \"%s\" 以及其依赖标签？"
```
Comment (ylc395) [inline CliClient/locales/zh_CN.po:903]: descendant tags 是后代标签的意思，不是依赖标签

File: CliClient/locales/zh_CN.po [hunk 2]
```diff
@@ -1778,7 +1778,7 @@ msgstr "统计数据"
 #: ElectronClient/gui/NoteContentPropertiesDialog.js:100
 #, javascript-format
 msgid "Read time: %s min"
-msgstr ""
+msgstr "读取时间： %s 分钟"
```
Comment (ylc395) [inline CliClient/locales/zh_CN.po:1781]: Read time 是“阅读所需时间”，不是读取时间

File: CliClient/locales/zh_CN.po (hunk 1 above)
Comment (ylc395) [inline CliClient/locales/zh_CN.po:903]: descendant tags 是后代标签的意思，不是依赖标签

File: CliClient/locales/zh_CN.po [hunk 3]
```diff
@@ -2778,17 +2778,16 @@ msgstr ""
 " %s"
 
 #: ReactNativeClient/lib/models/Tag.js:384
-#, fuzzy
 msgid "Cannot move tag to this location."
-msgstr "无法移动笔记本到该位置"
+msgstr "无法移动笔记本到该位置."
```
Comment (ylc395) [inline CliClient/locales/zh_CN.po:2782]: 是“无法移动标签到该位置.”

Comment (cyy5358) [reply CliClient/locales/zh_CN.po]: 上述错误已改正

**Comment by ylc395** [comment]
I think it's good to merge now.

**Comment by laurent22** [comment]
Thanks for the update!

**Pull request merged**